src/loc/CloseLocButton.tsx

```diff
@@ -137,6 +136,19 @@ export default function CloseLocButton(props: Props) {
         }
     }, [ closeState, setCloseState, accounts, axiosFactory, loc, navigate, props.protectionRequest, refreshRequests ]);
 
+    const isAcknowledgedByAtLeastVerifiedIssuer = useCallback((item: LocItem) => {
+        if(locState) {
+            return item.status === "ACKNOWLEDGED"
+                || (item.status === "PUBLISHED" && (item.acknowledgedByVerifiedIssuer || locState.isRequester(item.submitter) || locState.isOwner(item.submitter)));
+        } else {
+            return false;
+        }
+    }, [ locState ]);
+
+    const canAutoAck = useMemo(() => {
+        return locItems.length > 0 && locItems.find(item => !isAcknowledgedByAtLeastVerifiedIssuer(item)) === undefined;
+    }, [ locItems, isAcknowledgedByAtLeastVerifiedIssuer ]);
+
```

Comment:
I have the impression that this logic is kind of duplicated from the client. It would be better to expose this logic via the class [OpenLoc class](https://github.com/logion-network/logion-api/blob/0132cce4bc3ceeaf33489c7d49245db6420c1602/packages/client/src/Loc.ts#L1496), something like:
`
canClose(autoAck: boolean): boolean {}`

Reply:
Good point, I'll do that.